applications/google_rise_2016.md

```diff
@@ -0,0 +1,414 @@
+
+No
+
+## Additional Details
+
+### About the Organization
+
+Now help us learn a bit more about the overall organization that is applying for
+funding.
+
+**Mission Statement**
+
+Our mission is to provide every student, regardless of ethnicity, gender, or
+socioeconomic status, the opportunity to experience the joys of hacking because
+we believe coding is one of the most empowering things someone can learn. If
+we're able to accomplish this, not only does that mean a larger and more diverse
+technology workforce, but it means living in a more empowered world where more
+people create the change they want to see. And that is a world we want to live
+in
+
+**Brief description of the organization**
+
+Hack Club helps high school students start coding clubs. We provide prospective
+club leaders with open-source materials for starting clubs and with a supportive
+community of leaders around the globe.
+
+Our clubs have gone on to host hackathons with hundreds of participants, build
+games to destigmatize sex education, and bring coding education to traditionally
+underserved and underrepresented communities.
+
+**Organization Legal Status (if other please state)**
+
+Non-profit
+
+**Are you operating under a fiscal sponsor?**
+
+[X] Yes
+[ ] No
+
+**If you are operating under a fiscal sponsor, please complete the following
+fields.**
+
+**Name of fiscal sponsor**
+
+Mission Bit
+
+**Fiscal sponsor EIN or charitable registration identification number**
+
+46-0945785
+
+**Fiscal sponsor country**
+
+United States
+
+**Fiscal sponsor address 1**
+
+2448 Folsom St
+
+**Fiscal sponsor address 2**
+
+**Fiscal sponsor city**
+
+San Francisco
+
+**Fiscal sponsor state**
+
+California
+
+**Fiscal sponsor zip/postal code**
+
+94110-2620
+
+**Fiscal sponsor contact name**
+
+Tyson Daugherty
+
+**Fiscal sponsor phone**
+
+[INSERT_LATER]
+
+**Fiscal sponsor contact email address**
+
+[INSERT_LATER]
+
+**List of Board of Directors/Advisory Board members and affiliations (or provide
+a link)**
+
+_Please include names and affiliations of all Board members, particularly
+employees of Google._
+
+Advisory board:
+
+- Tom Preston-Werner, Co-Founder and former CEO of GitHub,
+  http://tom.preston-werner.com/
+
+**Organization social media channels**
+
+- Twitter: @starthackclub
+- Facebook: https://www.facebook.com/Hack-Club-741805665870458/
+
+**Number of staff employed by the organization**
+
+1
+
+**In what year was your organization founded?**
+
+2014
+
+**Number of youth reached by organization in 2015**
+
+1,500+
+
+**Demographics of youth reached by organization in 2015**
+
+_Please use approximate percentages to list the demographics of the youth that
+your organization served last year, such as age ranges, race/ethnicity, gender
+etc. For example, 25% boys, 75% girls._
+
+- 32% women (compare with 20% female AP CS test takers)
+- 37% minorities in tech (compare with 13% African American/Hispanic AP CS test
```

Comment:
Yea, I wrote this but I just pulled it off the website... anyone got better stats?